rllib/utils/replay_buffers/prioritized_episode_replay_buffer.py

```diff
@@ -259,8 +371,8 @@ def sample(
         next_observations = [[] for _ in range(batch_size_B)]
         actions = [[] for _ in range(batch_size_B)]
         rewards = [[] for _ in range(batch_size_B)]
-        is_terminated = [[False] * batch_length_T for _ in range(batch_size_B)]
-        is_truncated = [[False] * batch_length_T for _ in range(batch_size_B)]
+        is_terminated = [[False] for _ in range(batch_size_B)]
```

Comment:
Can you explain, why this fix? B/c a returned batch only has one terminated/truncated flag per batch row?

Reply:
Mainly because each batch row has only a single tuple. See my comment above.